src/styles/_academy.scss

```diff
@@ -518,3 +518,24 @@
     margin-left: 3px;
   }
 }
+
+.grading-overview-filterable-btns {
+  // Invisible border for backgroundless buttons
+  &:not(:has(> div)) {
+    margin-top: 1px;
+    border-bottom: 1px solid rgba(0, 0, 0, 0);
+  }
+
+  &:hover {
+    // Buttons with bg
+    > div > span {
+      // Use of contrast due to unknown background color
+      filter: contrast(0.9);
+    }
+
+    // Backgroundless buttons
+    &:not(:has(> div)) {
+      border-bottom-color: rgba(0, 0, 0, 0.5);
```

Comment:
Is there any particular reason we are using a bottom border, instead of `text-decoration: underline`?

Reply:
Nope, forgot text-decoration was a possible option, too used to borders. I've resolved all 3 comments in the follow up commit.